ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -70,3 +72,41 @@ def test_torch_cat(
         dim=unique_idx,
         out=None,
     )
+
+
+# permute
+@given(
+    dtype_values_axis=helpers.dtype_values_axis(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes))),
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    axis = helpers.get_axis(shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"), ret_tuple=True),
```

Comment:
it seems that there is one extra "," at the end of this line, is this an error?

Reply:
Also this line have some formatting issues:
ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:92:5: E122 continuation line missing indentation or outdented
ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:92:9: E251 unexpected spaces around keyword / parameter equals
ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:92:11: E251 unexpected spaces around keyword / parameter equals
ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:92:89: E501 line too long (109 > 88 characters)

Reply:
> it seems that there is one extra "," at the end of this line, is this an error?

```
    axis = helpers.get_axis(
        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"), 
        ret_tuple=True),
```
you mean the `,` after `ret_tuple`? `ret_tuple` is the arg I added to the helpers.get_axis function